crates/src/lib.rs

```diff
@@ -720,10 +720,10 @@ pub struct DriftClientBackend {
     pubsub_client: Arc<PubsubClient>,
     program_data: ProgramData,
     blockhash_subscriber: BlockhashSubscriber,
-    account_map: AccountMap,
-    perp_market_map: MarketMap<PerpMarket>,
-    spot_market_map: MarketMap<SpotMarket>,
-    oracle_map: OracleMap,
+    pub account_map: AccountMap,
+    pub perp_market_map: MarketMap<PerpMarket>,
+    pub spot_market_map: MarketMap<SpotMarket>,
+    pub oracle_map: OracleMap,
```

Comment:
feature flag these too
```suggestion
    pub account_map: AccountMap,
    pub perp_market_map: MarketMap<PerpMarket>,
    pub spot_market_map: MarketMap<SpotMarket>,
    pub oracle_map: OracleMap,
```